gui/default/syncthing/core/syncthingController.js

```diff
@@ -1661,6 +1661,20 @@ angular.module('syncthing.core')
             $http.post(urlbase + "/db/scan?folder=" + encodeURIComponent(folder));
         };
 
+        $scope.setAllFolderPause = function(pause) {
+            var folderListCache = $scope.folderList();
+
+            for (var i = 0; i < folderListCache.length; i++) {
+                var folder = folderListCache[i];
+                if (folder) {
```

Comment:
I don't know, in $scope.setFolderPause it also checks if the folder is valid.
I also don't have my name in the authors file.